frontend/components/vault/VaultLiquidationTransactionFlow.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+    <div>
+        <TextBlock title="Vault Liquidation transaction" />
+        <Alert v-if="vaultTransaction.state === 'liquidated'" show-icon type="warning">
+            <div slot="message">
+                <p>This vault has been liquidated into a collateral auction</p>
+                <div class="flex justify-end mt-2">
+                    <nuxt-link :to="vaultLink">
+                        <Button> View collateral auction {{ vaultTransaction.auctionId }} </Button>
+                    </nuxt-link>
+                </div>
+            </div>
+        </Alert>
+        <VaultLiquidationTransactionTable class="my-4" :vault-transaction="vaultTransaction" />
```

Comment:
Do you also need to show it if the vault is liquidated?

Reply:
It shows that the values are unknown. This is also what we do with the auctions.

Reply:
See my suggestion here: https://github.com/sidestream-tech/unified-auctions-ui/pull/454#issuecomment-1247681874